bridge/whatsappmulti/whatsapp.go

```diff
@@ -166,19 +172,14 @@ func (b *Bwhatsapp) Disconnect() error {
 func (b *Bwhatsapp) JoinChannel(channel config.ChannelInfo) error {
 	byJid := isGroupJid(channel.Name)
 
-	groups, err := b.wc.GetJoinedGroups()
```

Comment:
In reality 5 was not a hard limit on joined chats or anything.
Rather we were calling `GetJoinedGroups` too frequently and 5 happened to be the rate limit.
We really only care about this a single time per bridge so storing this in the bridge on create should not pose any compatibility issues I believe. 

This will improve bot boot times as get joined groups would add a delay to each channel joined.
And removed the need for multiple devices to work with many WhatsApp chats.